components/Common/LanguageDropDown/index.module.css

```diff
@@ -0,0 +1,34 @@
+.iconWrapper {
+  @apply h-[36px] 
+	  w-[36px] 
```

Comment:
I'm almost sure, we already have predefined variables for sizes? @canerakdas I remember you mentioned something about that?

Reply:
Yes, we can use `w-9` and `h-9` there its equal to  `2.25rem; /* 36px */`. 

@Rounak-stha  We prefer default classes because;

- Allows editing and formatting styles in a one-place
- Tailwind uses [rem for spacing](https://github.com/tailwindlabs/tailwindcss/blob/master/stubs/config.full.js#L815-L850), it would be annoying if this area remains fixed while the rest of the page is updated relative to the browser font size.